api-catalog-ui/frontend/src/reducers/authentication.reducer.jsx

```diff
@@ -19,6 +24,10 @@ function authenticationReducer(state = {}, action) {
         case userConstants.AUTHENTICATION_FAILURE:
             return {
                 error: action.error,
+                sessionOn: sessionDefaultState.sessionOn,
+                onCompleteHandling: function () {
+                    sessionDefaultState.sessionOn = false
+                }
             };
         case userConstants.USERS_LOGOUT_REQUEST:
```

Comment:
does it make sense to merge these cases?
case userConstants.USERS_LOGOUT_REQUEST:
case userConstants.USERS_LOGOUT_SUCCESS:

Reply:
No no, they are different actions, one for the request and one is dispatched when the logout was successful.